src/DynamoCore/DynamoCore.csproj

```diff
@@ -9,7 +9,7 @@
     <RootNamespace>Dynamo</RootNamespace>
     <AssemblyName>DynamoCore</AssemblyName>
     <UseVSHostingProcess>false</UseVSHostingProcess>
-    <DocumentationFile>$(OutputPath)\$(UICulture)\DynamoCore.XML</DocumentationFile>
+    <DocumentationFile>$(OutputPath)\DynamoCore.XML</DocumentationFile>
```

Comment:
does this change the generated nuget package structure?

Reply:
The path is the same as before
I think the diff viewer might not work properly.

Reply:
it is the same path as before